### ISSUE TITLE
Dynamic Dropdown doesn't announce search in progress

### ISSUE THREAD

**Issue by avernet** [orbeon/orbeon-forms#5767]
Once the user starts the search by typing with the dropdown field, screen readers should announce, that a search is currently being performed.

So far only the search string is read -> in the example below "a" "b" is read, however the word "Suche ..." is not mentioned. Therefore the user does not know, that a search is already being executed.

![image](https://user-images.githubusercontent.com/106961/232623879-9ac398bd-ab1e-49ae-b071-3142d488d16e.png)

[+1 from customer](https://3.basecamp.com/3600924/buckets/2016476/messages/6053282960)
